src/changelog/2024/07/flowfuse-assistant.md

```diff
@@ -0,0 +1,25 @@
+---
+title: The FlowFuse Assistant
+description: FlowFuse now allows selecting and comparing snapshots to visualize differences, facilitating easier understanding of changes between snapshots.
+date: 2024-07-05 13:00:00.0
+authors: ["stephen-mclaughlin"]
+tags:
+    - changelog
+---
+
+We are excited to announce the first release of The FlowFuse Assistant, a new plugin for your Node-RED instances designed to enhance your development experience.
+
+In this initial release, The FlowFuse Assistant can help you create function nodes for those times when a no-code solution is either not feasible or not possible.
+Additionally, it works directly inside the Function Node editor right where you need it.
+
+This is just the beginning. The FlowFuse Assistant will continue to evolve, with future updates bringing even more powerful features and enhancements.
+
+Starting from today, FlowFuse users can try this out on their instances.

```

Comment:
Will this be available to existing instances, new instances only, or is there something users need to do to upgrade and get the feature?

Reply:
Nick made a suggestion and has been committed.  Please re-review. (resolves comment)